pages/common/q.md

```diff
@@ -0,0 +1,23 @@
+# q
+
+> Execute SQL-like queries on .csv and .tsv files.
+
+- Query simple .csv file (with header for column name mapping):
+
+`q -d, -H "SELECT * from {{path/to/file}} where answer = 42"`
```

Comment:
Force-push to your own branch is ok, no worries. You can also just add more commits on top, and we'll squash them at the moment of merging (this is actually easier for you and better for code review, since we can see what changed between review rounds)